config/initializers/sorcery.rb

```diff
@@ -122,10 +122,10 @@
   # config.facebook.access_permissions = ["email", "publish_stream"]
   # config.facebook.display = "page"
   #
-  config.github.key = "#{Rails.application.secrets.sorcery_github_key}"
-  config.github.secret = "#{Rails.application.secrets.sorcery_github_secret}"
-  config.github.callback_url = "#{Rails.application.secrets.sorcery_github_callback_url}"
-  config.github.user_info_mapping = {:email => "name"}
+  config.github.key = '#{Rails.application.secrets.sorcery_github_key}'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.